packages/composites/augur/src/methods/createMarkets.ts

```diff
@@ -83,6 +83,8 @@ export const execute: ExecuteWithConfig<Config> = async (input, config) => {
   return Requester.success(input.id, {})
 }
 
+const transformSpecialNone = (val?: number) => val === 0.0001 ? undefined : val
```

Comment:
Could we add a JSdoc comment about when this happens?

Reply:
Yep, definitely